modules/ROOT/pages/installing-multiple-instances.adoc

```diff
@@ -0,0 +1,226 @@
+= Installing Multiple Instances of Runtime Fabric on a Single Cluster
+
+Installing multiple instances of Runtime Fabric enable you to share the same cluster among multiple Runtime Fabrics. This helps you to use resources efficiently.
+
+[NOTE]
+You must upgrade to the minimum version of agent to support multiple instances to be able to avail this feature.
+
+== Install Multiple Instances of Runtime Fabric
+
+To install multiple instances of Anypoint Runtime Fabric on a single BYOK (Bring Your Own Kubernetes) cluster, follow the next steps:
+
+. Create a Runtime Fabric using Runtime Manager.
+. Create a custom namespace to install the Runtime Fabric by running the following command:
++ 
+----
+kubectl create ns <rtf_namespace>
+----
+[start=3]
+. Create a Docker pull secret to pull the Runtime Fabric component images for the previously created namespace: 
++
+----
+kubectl create secret docker-registry <rtf_pull_secret> --namespace <rtf_namespace> --docker-server=<docker_registry_url> --docker-username=<docker_registry_username> --docker-password=<docker_ registry_password>
+----
+[start=4]
+. Add the Runtime Fabric Helm Repository: 
++
+----
+helm repo add <name> <helm_repo_url> --username <your_username> --password <your_password>
+----
+[start=5]
+. Optionally, configure the shared tenancy. Refer to the <<configure-namespaces,Configure Authorized Namespaces>> section for details. 
+. Download and configure the values.yaml file from the Anypoint Platform UI.
```

Comment:
```suggestion
. Download and configure the `values.yaml` file from the Anypoint Platform UI.
```

Reply:
Specifically, where would you download that?

Reply:
filenames are mono font: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-monospacefont

Reply:
Is it values.yml? or values.yaml?